source/Nuke.MSBuildTasks/ExternalFilesTask.cs

```diff
@@ -53,7 +53,6 @@ protected override bool ExecuteInner()
             return ExternalFiles
                 .SelectMany(GetExternalFilesInfo)
                 .Select(DownloadExternalFile)
-                .ToList()
```

Comment:
I'm pretty sure this is incorrect here.

Reply:
hmm I'm surely missing something 😅 
Can I ask why? Does it affect the `.All()` in the next line?

Reply:
`DownloadExternalFile` returns a `Task<T>` ... that `.ToList()` is required to trigger the parallelization, otherwise it will just sequentially check the returned results.

Reply:
That makes sense, thanks for the clarification.

Reply:
But there is no AsParallel() to start parallelization? Or am i missing something?